packages/xchain-util/src/chain.const.ts

```diff
@@ -1,15 +1,14 @@
 /**
  * Chains
- *
- * Based on definition in Thorchain `common`
- * https://gitlab.com/thorchain/thornode/-/blob/master/common/chain.go#L15-18
  */
 export const BNBChain = 'BNB'
 export const BTCChain = 'BTC'
 export const ETHChain = 'ETH'
 export const THORChain = 'THOR'
+export const CosmosChain = 'GAIA'
+export const PolkadotChain = 'POLKADOT'
```

Comment:
Don't we use `DOT` ([as discussed in Slack](https://thorchain.slack.com/archives/CPMV9FN80/p1606963852131100?thread_ts=1606909496.125400&cid=CPMV9FN80)) or do I miss something?

Reply:
We are now going with `POLKA`.`DOT` for polkadot. Just fixed and updated pr.